replica/database.hh

```diff
@@ -1171,6 +1173,12 @@ public:
     // Returns true if any of the sstables requries cleanup.
     bool requires_cleanup(const sstables::sstable_set& set) const;
 
+    // Takes snapshot of current storage state (includes memtable and sstables) from
+    // all compaction groups that overlap with a given token range. The output is
+    // a SSTable set that contains all the data (even if a subset was previously
```

Comment:
SSTable list, right? (it's confusing that it is implemented as a set)
Maybe better phrased as a "list of sstables" or a "set of sstables" so it won't be confused with sstable_set.

Reply:
> SSTable list, right? (it's confusing that it is implemented as a set) Maybe better phrased as a "list of sstables" or a "set of sstables" so it won't be confused with sstable_set.

yes, although I used sstable set as a conceptual term, I think it's better to say list of sstables to avoid confusion.

Reply:
thanks for your review.